src/leibniz.rs

```diff
@@ -8,18 +8,14 @@ fn main() {
     file.read_to_string(&mut contents)
         .expect("something went wrong reading the file");
 
-    let rounds = &contents.trim().parse::<i32>().unwrap();
+    let rounds = &contents.trim().parse::<u32>().unwrap() + 2;
```

Comment:
This is correct. In some languages it is considered unusual to use unsigned integers, but Rust programmers habitually use unsigned integers whenever negative values are actually irrelevant. This is partly spurred by the lack of integer promotion rules or special optimizations around signed integers: both kinds are treated alike.